scripts/activation_responses.py

```diff
@@ -0,0 +1,195 @@
+    print('Fill out the fields in these files then run ">> activation_response.py step1"')
+
+
+def step1():
+    config = ConfigParser.ConfigParser()
+    config.read(config_filename)
+
+    structured = config.getboolean('general', 'structured')
+    meshtal = config.get('step1', 'meshtal')
+    tally_num = config.getint('step1', 'tally_num')
+    flux_tag = config.get('step1', 'flux_tag')
+    decay_times = config.get('step2', 'decay_times').split(',')
+    geom = config.get('step1', 'geom')
+    reverse = config.getboolean('step1', 'reverse')
+    num_rays = config.getint('step1', 'num_rays')
+    grid = config.getboolean('step1', 'grid')
+    responses = config.get('general', 'responses').split(',')
+    wdr_file = config.get('general', 'wdr_file')
+
+    load(geom)
+
+    # get meshtal info from meshtal file
+    flux_mesh = resolve_mesh(meshtal, tally_num, flux_tag)
+
+    # create the cell_fracs array before irradiation_steup
+    if flux_mesh.structured:
+        cell_fracs = discretize_geom(flux_mesh, num_rays=num_rays, grid=grid)
+    else:
+        cell_fracs = discretize_geom(flux_mesh)
+
+    cell_mats = cell_materials(geom)
+    irradiation_setup(flux_mesh, cell_mats, cell_fracs, alara_params_filename,
+                      tally_num, num_rays=num_rays, grid=grid, reverse=reverse,
+                      output_mesh="activation_responses_step1.h5m",
+                      flux_tag=flux_tag, decay_times=decay_times,
+                      sub_voxel=False, responses=responses, wdr_file=wdr_file)
+
+    # create a blank mesh for step 2:
+    ves = list(flux_mesh.iter_ve())
+    for tag in flux_mesh.get_all_tags():
+        tag.delete()
+    flux_mesh.write_hdf5('blank_mesh.h5m')
+    print('The file blank_mesh.h5m has been saved to disk.')
+    print('Do not delete this file; it is needed by activation_responses.py step2.\n')
+    print('Decay_heat step1 complete, run ALARA with the command:')
+    print('>> alara alara_inp > output.txt')
+
+
+def step2():
+    config = ConfigParser.ConfigParser()
+    config.read(config_filename)
+    structured = config.getboolean('general', 'structured')
+    decay_times = config.get('step2', 'decay_times').split(',')
+    try:
+        alara_out = config.get('step2', 'alara_out')
+    except:
+        alara_out = "output.txt"
+    responses = config.get('general', 'responses').split(',')
+
+    for response in responses:
+        response_to_hdf5(alara_out, response)
+        tag_name = response
+        for i, dc in enumerate(decay_times):
+            print('Writing {0} for decay time: {1}'.format(response, dc))
+            mesh = Mesh(structured=structured, mesh='blank_mesh.h5m')
+            tags = {('TOTAL', dc): tag_name}
+            response_hdf5_to_mesh(mesh, ''.join([response, '.h5']), tags, response)
+            mesh.write_hdf5('{0}_{1}.h5m'.format(response, i+1))
+
+    print('Activation_response step2 complete.')
+
+
+def main():
+
```

Comment:
It is customary to put a docstring on this line ( in this case, describing the activities of the main function)
Also, PEP8 discourages unnecessary blank lines.